openfasoc/generators/temp-sense-gen/tools/parse_rpt.py

```diff
@@ -1,3 +1,8 @@
+import os, sys 
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+from common.check_gen_files import check_gen_files
+
 drc_filename = "flow/reports/sky130hd/tempsense/6_final_drc.rpt"
```

Comment:
TBH I don't know why we are considering the lvs and drc reports from this folder since we already have them in the work directory. Can we change this and keep everything aligned to a single convention?